solutions/usaco-265.mdx

```diff
@@ -0,0 +1,109 @@
+---
+id: usaco-265
+source: Old Gold
+title: The Cow Run
+author: Ryan Chou
+---
+ 
+[Official Analysis (C++)](http://www.usaco.org/current/data/sol_cowrun_mar13.html)
+
+## Explanation
+
+Since we'll always want to pick up a cow when we can, the list of cows that we've visited will form a subarray in the sorted list of cows.
```

Comment:
> when we can

wdym "when we can"?
how about
"when we go from one point to another, it's optimal to pick up every cow along the way"

Reply:
okeee